loadtest/report/report.go

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) 2019-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package report
+
+import (
+	"encoding/json"
+	"io/ioutil"
+
+	"github.com/mattermost/mattermost-load-test-ng/coordinator/performance/prometheus"
+)
+
+// Generator is used to generate load test reports.
+type Generator struct {
+	label  string
+	helper *prometheus.Helper
+}
```

Comment:
Is there a reason `Generator` is defined in `report.go` and `Report` in `generate.go`?

Reply:
The filename report.go is actually not perfect, and needs to be something else. Essentially report.go returns a generator. Therefore the `New` function and its struct is there. And generate.go generates a report. So the `Report` struct is there, because its easier to look at the fields of the report from where its generated.

Any suggestions for better names?

Reply:
Maybe we don't really need the two files. Everything in `generate.go` receives `Generator`. 
It might be just fine to have both `Report` and `Generator` there.

Reply:
Ok